documents/openapi/openapi.yaml

```diff
@@ -1191,6 +1191,73 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/NotificationStatusResponse'
+              examples:
+                sms:
+                  value:
+                    billing_code: string
+                    body: string
+                    completed_at: string
+                    created_at: string
```

Comment:
These are `$date-times`,  so the examples should also be `$date-time`s (see the GET response for `/service/{service_id}/api-keys/{api_key_id}` if you need an example).

Reply:
@cris-oddball you meant for `created_at` and `completed_at` right? Not `created_by_name` I suppose.

Reply:
correct, that is the comment is for lines 1199 and 1200.